.env

```diff
@@ -1,16 +1,33 @@
-NEXT_PUBLIC_PROD_CONTAINER_API_URL="http://prod:3001"
-NEXT_PUBLIC_DEVELOPMENT_API_URL="http://localhost:3000"
-NEXT_PUBLIC_MAIN_LANGUAGE="uk"
-NEXT_PUBLIC_FALLBACK_LANGUAGE="en"
-NEXT_PUBLIC_GRAPHQL_API_URL="http://localhost:4000/graphql"
-GRAPHQL_API_PATH="graphql"
-HEALTH_CHECK_PATH="health"
-NEXT_PUBLIC_API_URL="https://yourserver.io/api/"
-NEXT_PUBLIC_VILNACRM_GMAIL="info@vilnacrm.com"
-NEXT_PUBLIC_VILNACRM_PRIVACY_POLICY_URL="https://github.com/VilnaCRM-Org"
-NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME="no-aws-header-name"
-NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_VALUE="no-aws-header-value"
-GRAPHQL_SCHEMA_URL="https://raw.githubusercontent.com/VilnaCRM-Org/user-service/v2.4.1/.github/graphql-spec/spec"
-GRAPHQL_LOG_LEVEL='info'
+LOCAL_HOST=localhost
```

Comment:
Please use WEBSITE_DOMAIN environment variable name instead and check all localhost strings in the code, we should use environment variable instead of hardcoding